backend/v1/dependencies/database/database_connector.py

```diff
@@ -13,6 +13,8 @@ class Connector:
 
     FISH_SPECIES_IDENTIFIER_FIELD_NAME = 'species_name'
 
+    AQUARIUMS_COLLECTION = 'aquariums'
```

Comment:
Move the AQUARIUMS_COLLECTION under USERS_COLLECTION to prevent mess in the code

Reply:
todo in next comment